tools/batch/docker/Dockerfile

```diff
@@ -22,6 +22,16 @@ FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04
       rm -rf /var/lib/apt/lists/*
 
  RUN pip3 install --upgrade pip && pip3 install awscli && pip3 install --pre 'mxnet-cu102' -f https://dist.mxnet.io/python
+
+ RUN ln -s /usr/bin/python3 /usr/bin/python && \
+    ln -s /usr/bin/pip3 /usr/bin/pip
```

Comment:
Should not do with `ln`.

Reply:
I think you may try to use `update-alternatives`.

Reply:
like `update-alternatives sudo update-alternatives --install /usr/bin/python python /usr/bin/python3` ?

Reply:
It may be better to revise the script to use `python3` as it improves portability

Reply:
After checking the TF dockerfile  https://github.com/tensorflow/tensorflow/blob/1e6fa32dfe9d3f1631ed6218985426d3f6ca4860/tensorflow/tools/dockerfiles/dockerfiles/devel-gpu-jupyter.Dockerfile#L111 , the correct approach might be to use `ln -s` but create the link in `/usr/local/bin/python`.